lib/core/watch.ts

```diff
@@ -12,6 +13,10 @@ import { ConfigOptions } from "./types";
  * @param options the CLI options
  */
 export const watch = (pattern: string, options: ConfigOptions): void => {
+  // This is called so that we print a warning instead if no files matched the
+  // pattern
```

Comment:
nit: I think you did a good job naming this function well and this comment is slightly wrong (if 0 or 1 file is matched) which indicates to me it's best to remove it since it's already out of date/redundant and allow readers to instead read the description of this function which is accurate since it's in the same file as the function which is easy to keep up to date if it's ever updated